website/forms/php/filters/general/set-locale.md

```diff
@@ -3,4 +3,11 @@ id: set-locale
 title: Set locale
 ---
 
-*Coming soon*
+This filter can be used to change the value of current locale. By default, WordPress sets the locale in the admin to `en_US`. With this filter it can be changed to any locale (e.g. when using a multi-language plugin).
```

Comment:
```suggestion
This filter can be used to change the value of current locale. By default, WordPress sets the locale in the admin to `en_US`. With this filter it can be changed to any locale (e.g. when using a multilingual plugin).
```